interface/orders/procedure_provider_edit.php

```diff
@@ -341,6 +341,7 @@ class='form-control' />
                                             'SFTP' => xl('SFTP'),
                                             'FS' => xl('Local Filesystem'),
                                             'WS' => xl('Web Service'),
+                                            'DORN' => xl('Dorn'),
```

Comment:
Probably need to use an event listener here.  If the module is disabled... this option shouldn't show up.